### PR TITLE
Create the “Piracy” command

### PR DESCRIPTION
**Pull request by 2tmb2**
Creates the “piracy” command which links directly to Eip’s page on piracy, located here: https://3ds.eiphax.tech/piracy

### CODE REVIEW

**Comment by 2tmb2** [comment]
Wasn’t sure what to do for the thumbnail so I used one that was previously used. That is probably going to change and should probably change, I just wasn’t sure how to change It or what to change it to.

File: cogs/assistance.py [hunk 1]
```diff
@@ -275,7 +275,17 @@ async def cfwuses(self, ctx, console=""):
                         - Dump your Wii U game discs to a format that can be installed on your internal or external Wii U storage drive.
                     """))
             await ctx.send(embed=embed)
-
+                               
+    @commands.command(aliases = ["pirate3", "whatisp"])
+    async def piracy(self, ctx):
+        """Links to Eip's page on piracy"""
+        embed = discord.Embed(title="Eip's Piracy Page", color=discord.Color(0xCE181E))
+        embed.set_author(Name="eip", url = "https://3ds.eiphax.tech/piracy")
```
Comment (bleck9999) [inline cogs/assistance.py:283]: Name should be all lowercase

File: cogs/assistance.py [hunk 2]
```diff
@@ -275,7 +275,17 @@ async def cfwuses(self, ctx, console=""):
...
+        embed.set__thumbnail(url="https://nintendohomebrew.com/pics/nhplai.png")
```
Comment (bleck9999) [inline cogs/assistance.py:284]: Should have only one underscore

**Review by bleck9999 — CHANGES_REQUESTED** [review]

File: cogs/assistance.py (hunk 1 above)
Comment (2tmb2) [reply cogs/assistance.py]: Fixed

File: cogs/assistance.py (hunk 2 above)
Comment (2tmb2) [reply cogs/assistance.py]: Fixed

**Comment by pbanj** [comment]
glazy told me to

**Comment by GlaZedBelmont** [comment]
to be accurate, we made an alternative that was merged in https://github.com/nh-server/Kurisu/commit/a3c9881dd3af324520ee7fe6e51c1fedf0c9a624

**Pull request closed without merging**